app/gui/src/model/execution_context/synchronized.rs

```diff
@@ -329,6 +329,21 @@ impl model::execution_context::API for ExecutionContext {
         }
         .boxed_local()
     }
+
+    fn start_execution(&self) -> BoxFuture<FallibleResult> {
+        async move {
+            self.language_server
+                .client
+                .recompute(
+                    &self.id,
+                    &language_server::InvalidatedExpressions::All,
+                    &Some(ExecutionEnvironment::Live),
+                )
+                .await?;
+            Ok(())
+        }
+        .boxed_local()
```

Comment:
We should also turn on "read-only" flag on controllers, implemented by @vitvakatu AFAIK.

Reply:
That should already happen through the double click, which emits the appropriate FRP event from the graph editor.